persistence/sql/migrations/sql/20221109114303000000_add_courier_send_attempts_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE courier_message_dispatches (
+  dispatch_id UUID PRIMARY KEY,
+  message_id UUID NOT NULL,
+  status TEXT NOT NULL,
```

Comment:
Set this to a maximum length

Reply:
The `TEXT` type doesn't have a maximum length. Its specification is ["a variable text type of unlimited length"](https://www.postgresql.org/docs/current/datatype-character.html#:~:text=variable%20unlimited%20length). We could change this to a `VARCHAR` and provide a maximum length, but since we don't know specifically what the error string could be I'd prefer `TEXT` here.

Reply:
Nvm, though you were talking about the "error" field. 

This of course can be a char/varchar, sorry for the noise.